ngx-fudis/projects/ngx-fudis/src/lib/components/description-list/description-list.stories.ts

```diff
@@ -117,11 +117,11 @@ const NestedDescriptionListsTemplate: StoryFn<DescriptionListComponent> = (
     <fudis-heading [level]="2" [variant]="'md'"
       >Nested Description Lists with Indiana Jones Movies</fudis-heading
     >
-    <fudis-dl class="fudis-mt-sm" [disableGrid]="disableGrid">
+    <fudis-dl class="fudis-mt-sm" [disableGrid]="disableGrid" [classes]="['fudis-mb-none']">
       <fudis-dl-item>
         <fudis-dt [contentText]="'Raiders of the Lost Ark'"></fudis-dt>
         <fudis-dd>
-          <fudis-dl [variant]="'compact'">
+          <fudis-dl [variant]="'compact'" [classes]="['fudis-mt-none fudis-mb-none']">
```

Comment:
In general I think this is a bad pattern.
Browser adds default styles (margins) to native `<dl>` element which we used to override with Grid's margin-top and margin-bottom values. I think we still want to override those values coming from the browser, so some king of logic for that should be added where `[classes]` input's values will then override the initial none values.